Engine/Engine/Core/CYBLogger.cpp

```diff
@@ -79,77 +79,64 @@ CYB::Engine::Logger::Logger(API::Logger& AEmergencyLogger) :
 #endif
 	)
 {
-	auto InitEntry(static_cast<Allocator&>(Context::GetContext().FAllocator).RawObject<LogEntry>());
-	InitEntry->FLevel = Level::INFO;
-	InitEntry->FMessage = FormatLogMessage(API::String::Static(u8"CyberEngineMkIII logger started..."), Level::INFO);
-	InitEntry->FNext = nullptr;
+	FQueueHead.reset(static_cast<Allocator&>(Context::GetContext().FAllocator).RawObject<LogEntry>());
+	FQueueHead->FLevel = Level::INFO;
+	FQueueHead->FMessage = FormatLogMessage(API::String::Static(u8"CyberEngineMkIII logger started..."), Level::INFO);
 
-	FQueueHead = InitEntry;
-	FQueueTail = InitEntry;
+	FQueueTail = FQueueHead.get();
```

Comment:
Remove the initializations of FQueueHead, FQueueTail, and FThread from the initializer list